Tools/TestWebKitAPI/Tests/WebKitCocoa/HSTS.mm

```diff
@@ -91,6 +91,10 @@ static HTTPServer hstsServer()
     EXPECT_WK_STREQ(webView.get().URL.absoluteString, "https://example.com/");
 }
 
+// FIXME: Re-enable after webkit.org/b/241342 is resolved
+#if (PLATFORM(IOS))
+TEST(HSTS, DISABLED_ThirdParty)
+#else
 TEST(HSTS, ThirdParty)
```

Comment:
Your `#endif` should go beneath this line.

Reply:
Thank you.  I'll fix.